### PR TITLE
zh-trans: add docs/setup/best-practices/node-conformance.md

### PR DESCRIPTION
**Pull request by SataQiu**

### CODE REVIEW

File: content/zh/docs/setup/best-practices/node-conformance.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
+---
+reviewers:
+- Random-Liu
+title: 节点设置校验
```
Comment (chenrui333) [inline content/zh/docs/setup/best-practices/node-conformance.md]:
```suggestion
title: 校验节点设置
```

File: content/zh/docs/setup/best-practices/node-conformance.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+  Arch  |       Image       |
+--------|:-----------------:|
+ amd64  |  node-test-amd64  |
+  arm   |    node-test-arm  |
+ arm64  |  node-test-arm64  |
+
+<!--
+## Running Selected Test
+-->
+## 运行选定的测试
```
Comment (chenrui333) [inline content/zh/docs/setup/best-practices/node-conformance.md]:
```suggestion
## 运行特定的测试
```

File: content/zh/docs/setup/best-practices/node-conformance.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+sudo docker run -it --rm --privileged --net=host \
+  -v /:/rootfs:ro -v $CONFIG_DIR:$CONFIG_DIR -v $LOG_DIR:/var/result \
+  -e SKIP=MirrorPod \ # Run all conformance tests but skip MirrorPod test
+  k8s.gcr.io/node-test:0.2
+```
+
+<!--
+Node conformance test is a containerized version of [node e2e test](https://github.com/kubernetes/community/blob/{{< param "githubbranch" >}}/contributors/devel/e2e-node-tests.md).
+-->
+节点一致性测试是[节点 e2e 测试](https://github.com/kubernetes/community/blob/{{< param "githubbranch" >}}/contributors/devel/e2e-node-tests.md)的容器化版本。
```
Comment (chenrui333) [inline content/zh/docs/setup/best-practices/node-conformance.md]: e2e = end to end, 端到端测试

```suggestion
节点一致性测试是[节点端到端测试](https://github.com/kubernetes/community/blob/{{< param "githubbranch" >}}/contributors/devel/e2e-node-tests.md)的容器化版本。
```

File: content/zh/docs/setup/best-practices/node-conformance.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+<!--
+By default, it runs all conformance tests.
+-->
+默认情况下，它会运行所有一致性测试。
+
+<!--
+Theoretically, you can run any node e2e test if you configure the container andmount required volumes properly. But **it is strongly recommended to only run conformance test**, because it requires much more complex configuration to run non-conformance test.
+-->
+理论上，只要合理地配置容器和挂载所需的卷，就可以运行任何的节点 e2e 测试用例。 但是这里**强烈建议只运行一致性测试**，因为运行非一致性测试需要很多复杂的配置。
```
Comment (chenrui333) [inline content/zh/docs/setup/best-practices/node-conformance.md]:
```suggestion
理论上，只要合理地配置容器和挂载所需的卷，就可以运行任何的节点端到端测试用例。 但是这里**强烈建议只运行一致性测试**，因为运行非一致性测试需要很多复杂的配置。
```

File: content/zh/docs/setup/best-practices/node-conformance.md [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+<!--
+* The test leaves some docker images on the node, including the node conformance
+  test image and images of containers used in the functionality
+  test.
+* The test leaves dead containers on the node. These containers are created
+  during the functionality test.
+-->
+
+* 测试会在节点上遗留一些 Docker 镜像， 包括节点一致性测试本身的镜像和功能测试相关的镜像。
+* 测试会在节点上遗留一些死的容器。这些容器是在功能测试的过程中创建的。
```
Comment (chenrui333) [inline content/zh/docs/setup/best-practices/node-conformance.md]:
```suggestion
* 测试会在节点上遗留一些废用的容器。这些容器是在功能测试的过程中创建的。
```

**Comment by SataQiu** [comment]
Thanks @chenrui333 Updated :smile:

**Pull request merged**